.github/workflows/pr.yml

```diff
@@ -20,13 +20,19 @@ concurrency:
 jobs:
   build:
     uses: ./.github/workflows/build-full.yml
+  doc:
+    if: ${{contains(github.head_ref, 'release-v') || contains(github.head_ref, 'doc-')}}
+    uses: ./.github/workflows/doc.yml
```

Comment:
Is there anything that can break the doc build?  If so we ought to build it for each PR, it not this is fine.

Reply:
> I think a follow-up could enforce it at every pull request? 

Sure

> (But we definitely don't want to upload at every pull.)

I completely agree.